pkg/build/rpm.go

```diff
@@ -0,0 +1,50 @@
+package build
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+func (b *Builder) getRPMFileNames() ([]string, error) {
+	var rpmFileNames []string
+	rpmSourceDir := filepath.Join(b.buildConfig.ImageConfigDir, "rpms")
```

Comment:
We already do this in `copyRPMs()` so we can pass the directory as an argument to this function.